src/lib/Scenes/Show/Screens/Detail.tsx

```diff
@@ -121,6 +153,10 @@ export const DetailContainer = createFragmentContainer(
         city
         state
         postal_code
+        displayDaySchedules {
```

Comment:
We abstracted where the data for this field is coming from in metaphysics right?
(Structured / unstructured data)

Reply:
@anandaroop added a field to `Location` for unstructured text data, I'll create a follow up ticket to make an abstracted `hoursDisplay` field that pulls from either this field or the unstructured `Fair.hours` or `Location.day_schedule_text`.